serve.js

```diff
@@ -40,6 +46,12 @@ const command = async (specFile, cmd) => {
         verbose
     });
 
+    let port = cmd.port;
```

Comment:
The cli interface should probably override given config file values, not the other way around. 

This does make things a bit tricky trying to do it in the command files, because `cmd.port` has a default value, [set in speccy.js](https://github.com/wework/speccy/blob/master/speccy.js#L44)

This means that instead of just doing something simple like this:

```
const port = config.get('serve.port') || cmd.port;
```

... we'd need to use `config.get()` in speccy.js and use it to define the default values.

That might look like this in speccy.js:

```
.option('-p, --port [value]', 'port on which the server will listen', config.get('serve.port') || 5000)
```

This means the CLI option is used if provided, then serve.port is inspected. If no value is set, it'll use 5000 as the default port! 

Similar stuff will have to be done for the other options too.